### PR TITLE
Add notes about XML Serialization changes

### PR DESCRIPTION
**Pull request by WilliamBZA**
I'm not 100% happy with the wording around the backwards compat

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Connects to https://github.com/Particular/NServiceBus/pull/3305

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -487,4 +487,10 @@ To re-add this header to outgoing messages a [mutator](/nservicebus/pipeline/mes
...
 
-Another option is to use your own custom header as illustrated in [Appending username using headers](/samples/username-header/) sample.
\ No newline at end of file
+Another option is to use your own custom header as illustrated in [Appending username using headers](/samples/username-header/) sample.
+
+## Standardized XML Serialization
+
+Null types within the XML serializer now conforms to the [W3C Specification](https://www.w3.org/TR/xmlschema-1/#xsi_nil) by using the `xsi:nil="true"` attribute.
+
+This change is backwards compatible and will not impact on communication between older versions of endpoints and newer versions. Older versions will be able to communicate with newer versions and vice versa.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: >  will not impact on communication between older versions of endpoints and newer versions

perhaps

> will have no impact on

or 

> will not impact communications

**Comment by SimonCropp** [comment]
@WilliamBZA this needs some text about external integration scenarios. ie we know it works for our other code. but of people have made assumption about the older format when sending or receiving in integration cases.

@danielmarbach cc

**Comment by danielmarbach** [comment]
@simoncropp I discussed this offline with william yesterday

**Comment by WilliamBZA** [comment]
Details and examples added

**Comment by SimonCropp** [comment]
@WilliamBZA this looks good to me. but needs a rebase. happy for me to merge manually? or would u prefer to handle the conflict yourself?

**Comment by WilliamBZA** [comment]
What merge conflicts?

**Comment by SimonCropp** [comment]
@WilliamBZA the PR says there are conflicts

**Comment by WilliamBZA** [comment]
Weird... didn't show any conflicts a couple minutes ago. I'll fix and update

**Comment by WilliamBZA** [comment]
Rebased

**Pull request merged**